src/Splitters/HtmlSplitter.php

```diff
@@ -23,29 +24,117 @@ class HtmlSplitter implements SplitterContract
      *
      * @var string[]
      */
-    protected $tags = [
+    protected $acceptedNodes = [
         'h1',
         'h2',
         'h3',
         'h4',
         'h5',
+        'h6',
         'p',
     ];
 
     /**
      * Creates a new instance of the class.
      *
-     * @param array $tags
+     * @param array $acceptedNodes
      *
      * @return void
      */
-    public function __construct(array $tags = null)
+    public function __construct(array $acceptedNodes = null)
     {
-        if ($tags !== null) {
-            $this->tags = $tags;
+        if ($acceptedNodes !== null) {
+            $this->acceptedNodes = $acceptedNodes;
         }
     }
 
+    /**
+     * Find it's value in $acceptedNodes.
+     *
+     * @param array $object
+     *
+     * @return int
+     */
+    public function findValue($object): int
```

Comment:
Missing type hint in in object, please review the pr to check if type hints are missing.